orc8r/gateway/python/magma/magmad/tests/collector_tests.py

```diff
@@ -171,9 +176,152 @@ def test_collect_start_time(self):
         mock.exception.side_effect = [False]
         try:
             self._collector.collect_done('test', mock)
-        except Exception:   # pylint: disable=broad-except
+        except Exception:  # pylint: disable=broad-except
             self.fail("Collection with empty metric should not have failed")
 
+    def test_counter_to_proto(self):
+
+        test_counter = prometheus_client.core.CounterMetricFamily("test", "",
```

Comment:
nit: extra whitespace above this line and break line before the first arg for aesthetics. apply below